tests/test_workflow.py

```diff
@@ -24,15 +24,45 @@ def test_full_workflow():
                                               "training_sig_files.txt"])))
     # one of the signature files
     expected_files.extend(list(map(lambda x: os.path.join(data_dir, intermediate_dir, "signatures", x), ["04212e93c2172d4df49dc5d8c2973d8b.sig.gz"])))
+    # In testdata/
```

Comment:
I deleted these as they were useful for my construction of the expected files, but just clutter things. Perhaps a `git pull origin main` was missing?

Reply:
please feel free to change any code in `test_workflow.py`. What I did is to make it adapt to the new change. So, I don't want to make any big changes for what you wrote.